locales-he-IL.xml

```diff
@@ -57,28 +57,31 @@
       <multiple>refs.</multiple>
     </term>
     <term name="retrieved">אוחזר</term>
-    <term name="scale">scale</term>
+    <term name="scale">קנ"מ</term>
     <term name="version">גירסה</term>
 
     <!-- ANNO DOMINI; BEFORE CHRIST -->
     <term name="ad">לספירה</term>
     <term name="bc">לפני הספירה</term>
 
     <!-- PUNCTUATION -->
-    <term name="open-quote">“</term>
-    <term name="close-quote">”</term>
-    <term name="open-inner-quote">‘</term>
-    <term name="close-inner-quote">’</term>
-    <term name="page-range-delimiter">–</term>
+    <term name="open-quote">"</term>
+    <term name="close-quote">"</term>
+    <term name="open-inner-quote">'</term>
+    <term name="close-inner-quote">'</term>
+    <term name="page-range-delimiter">-</term>
 
     <!-- ORDINALS -->
-    <term name="ordinal">th</term>
-    <term name="ordinal-01">st</term>
-    <term name="ordinal-02">nd</term>
-    <term name="ordinal-03">rd</term>
-    <term name="ordinal-11">th</term>
-    <term name="ordinal-12">th</term>
-    <term name="ordinal-13">th</term>
+    <term name="ordinal-01">1</term>
+    <term name="ordinal-02">2</term>
+    <term name="ordinal-03">3</term>
+    <term name="ordinal-04">4</term>
+    <term name="ordinal-05">5</term>
+    <term name="ordinal-06">6</term>
+    <term name="ordinal-07">7</term>
+    <term name="ordinal-08">8</term>
+    <term name="ordinal-09">9</term>
+    <term name="ordinal-10">10</term>
```

Comment:
Incorrect use of ordinal terms.

Reply:
If I remember correctly, the problem here that you don't use suffixes for this in Hebrew, but you'll always have
Also: in Hebrew to ordinal appears after---no before---the word that is counted: mahadurah rishonah => first edition; mahadurah shlishit => third edition